analytics/reports/orders.md

```diff
@@ -0,0 +1,567 @@
+                     WHEN oa.ATTR_NAME = 'NETSUITE_ORDER_EXPORTED' THEN oa.ATTR_VALUE
+                 END) AS NETSUITE_ORDER_EXPORTED
+      FROM order_attribute oa
+      WHERE oa.ATTR_NAME IN ('SIGNIFYD_APPROVED',
+                             'NETSUITE_ORDER_EXPORTED')
+      GROUP BY oa.ORDER_ID) temp1
+   JOIN order_header oh ON temp1.ORDER_ID = oh.ORDER_ID
+   JOIN order_role or2 ON temp1.ORDER_ID = or2.ORDER_ID
+   AND or2.ROLE_TYPE_ID = 'SHIP_TO_CUSTOMER'
+   AND (or2.THRU_DATE IS NULL
+        OR or2.THRU_DATE > NOW())
+   LEFT JOIN party_identification pi2 ON or2.PARTY_ID = pi2.party_id
+   AND pi2.PARTY_IDENTIFICATION_TYPE_ID = 'NETSUITE_CUSTOMER_ID'
+   WHERE oh.STATUS_ID = 'ORDER_CREATED'
+     AND oh.SALES_CHANNEL_ENUM_ID NOT IN ('LOOP_EXCH')
+     AND (SIGNIFYD_APPROVED IS NULL
+          OR NETSUITE_ORDER_EXPORTED IS NULL)
+   ORDER BY oh.ENTRY_DATE) AS virtual_table
+ORDER BY `ENTRY_DATE` ASC
+LIMIT 1000;
+```
+
+</details>
+
+### Query Logic
+
+**Data Selection:** The SQL query starts by selecting specific data fields relevant to the report. This typically includes information about orders and their attributes.
+
+**Defining Missing Attributes:** The primary objective of the report is to identify orders with missing attributes. Therefore, criteria are established to filter the data and include only orders where certain attributes are absent or null.
+
+**Joining Relevant Tables:** To compile a comprehensive dataset, the SQL query joins tables containing order information and attributes. This typically involves joining the `order_header` table with the `order_attribute` table based on a common field like `ORDER_ID`.
+
+**Selecting Necessary Information:** From the joined tables, the query selects columns related to order attributes and any other relevant information needed for analysis or reporting.
+
+**Filtering Data:** Conditions are applied to filter the data and include only orders with missing attributes. This might involve checking if certain attribute values are null or absent in the dataset.
+
+**Grouping Data:** The query groups the selected data based on certain criteria, such as order ID or attribute type. Grouping data helps summarize and organize it for analysis.
+
+**Counting Missing Attributes:** Within each group, the query counts the number of orders with missing attributes. This provides insights into the frequency and distribution of missing attributes across orders.
+
+## HotWax Order Count Report
+
+The HotWax Order Count Report offers a daily snapshot of new orders in the Order Management System (OMS), providing a quick reference for monitoring overall order activity and facilitating proactive decision-making. In the Shopify context, this report ensures seamless synchronization by comparing daily order counts. It serves as a vital tool for merchants, quickly identifying and addressing any discrepancies in the synchronization process, ensuring accurate and efficient order fulfillment.
```

Comment:
i don't think this helps in identifying discrepancies. this gives an overview of all the order counts recorder in HC

Reply:
I copied this description from user manual. Let me know if you still want me to change.